.github/workflows/ci.yml

```diff
@@ -1,7 +1,11 @@
 ---
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches: ["main"]
```

Comment:
Oh, this means that now the jobs aren't automatically triggered when the merge queue starts, so we'll need to also add "merge_group" here.